devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -113,6 +113,12 @@ def refresh() {
     zigbee.onOffRefresh() + zigbee.levelRefresh()
 }
 
+def installed() {
+    if ((device.getDataValue("manufacturer") == "MRVL") && (device.getDataValue("model") == "MZ100")) {
```

Comment:
Installed is only called once. When the DTH is changed, the updated method is called. But i will add the check if something goes wrong :)